python/family-relationship/family-tree-first-script.py

```diff
@@ -1,146 +1,9 @@
-import pprint
+import json
+import collections
+
+with open('D:/projects/json/training/data/family-tree.json') as jsonFile:
```

Comment:
you don't need to give it an absolute path! so if I copy this project on my local machine, it won't be executed, because I don't have a drive D or the same path as you! always use relative path not absolute path!
you are also exposing the data structure of your own machine which is a security issue!

absolute path: 'D:/projects/....'
relative path: '/data/family-tree.json' this is relative to your computer or anyone who clone this project, but absolute path is only on your computer not anyone else!